banking/src/main/java/rocks/kata/Account.java

```diff
@@ -0,0 +1,61 @@
+package rocks.kata;
+
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.concurrent.atomic.LongAdder;
+
+import rocks.kata.AccountOperation.OperationType;
+import rocks.kata.DatabaseClasses.AccountOperationsDatabase;
+
+public class Account {
+
+    private static final LongAdder numberOfAccounts = new LongAdder();
```

Comment:
https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

Reply:
But Jonarzz, "If any of the instance's observable state can change, it is not a constant." and this indeed changes. Może ten final zbędny, skoro to zmienna prywatna i nikt przypadkowo jej nie zmieni?

Reply:
Słusznie! Tak jest dobrze, mi się włączył w głowie SonarQube "static final => WIELKIE_LITERY" xD Tak jest git, zostaw jak jest.